chutney/action-impl/src/main/java/com/chutneytesting/action/function/MathFunctions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.action.function;
+
+import com.chutneytesting.action.spi.SpelFunction;
+
+public class MathFunctions {
+
+    @SpelFunction
+    public static Number abs(Number a) {
+        String numberType = a.getClass().getSimpleName();
+
+        return switch (numberType) {
+            case "Double" -> Math.abs(a.doubleValue());
+            case "Float" -> Math.abs(a.floatValue());
+            case "Integer" -> Math.abs(a.intValue());
+            case "Long" -> Math.abs(a.longValue());
+            default -> throw new IllegalArgumentException("Unexpected type: " + numberType);
+        };
+    }
+
+    @SpelFunction
+    public static Number min(Number a, Number b) {
+        String numberType = a.getClass().getSimpleName();
+
+        if (!numberType.equals(b.getClass().getSimpleName())) {
```

Comment:
may be we want to be able to comapre `1` and `1.5` for example.
To be discussed